scss/_card.scss

```diff
@@ -16,6 +16,7 @@
   --#{$prefix}card-cap-padding-x: #{$card-cap-padding-x};
   --#{$prefix}card-cap-bg: #{$card-cap-bg};
   --#{$prefix}card-cap-color: #{$card-cap-color};
+  --#{$prefix}card-title-color: #{$card-title-color};
```

Comment:
Could you please regroup it with the other `--#{$prefix}card-title-*` in the file?

Reply:
Fixed in https://github.com/twbs/bootstrap/pull/36979/commits/41419dfab6f4b22e0b1970700727ffa21525e611.